Layer/Pdo/Pdo.php

```diff
@@ -194,7 +194,12 @@ public function lastInsertId($name = null)
     public function prepare($statement, array $options = [])
     {
         if (!isset($options[\PDO::ATTR_CURSOR])) {
-            $options[\PDO::ATTR_CURSOR] = \PDO::CURSOR_SCROLL;
+            try {
+                $this->getConnection()->getAttribute(\PDO::ATTR_CURSOR);
+                $options[\PDO::ATTR_CURSOR] = \PDO::CURSOR_SCROLL;
```

Comment:
Did you try with this branch ?

I had the same error as you and this patch fix it (with sqlite3 on win10 x64)